### PR TITLE
Feature/exclude archives

### PR DESCRIPTION
**Pull request by PaysPlat**
Hi, 
I have asked in issue [issue 4](https://github.com/kewisch/quickmove-extension/issues/4) an option to exclude Archives folder.

So I have some time this week-end and I would try to implement by myself.
So it is done and functional.
I'm not sure if isSpecialFolder method is the optimal one.

I also suggest french localization for your extension.

Thank you for your time to review my contribution.

See you.

PaysPlat

### CODE REVIEW

File: src/content/quickmove.js [hunk 1]
```diff
@@ -253,16 +255,21 @@ var quickmove = (function() {
        * to iterate through any sub-sub-folders.
        *
        * @param aFolder  the folder to check
+       * @param excludeArchives  if Archives folder must be excluded
        */
-      function processFolder(aFolder) {
+      function processFolder(aFolder, excludeArchives) {
+         if (excludeArchives 
+          && aFolder.isSpecialFolder(0x00004000, false)) {
```
Comment (kewisch) [inline src/content/quickmove.js:262]: Instead of the number I think you can use `Ci.nsMsgFolderFlags.Archive`

**Review by kewisch — CHANGES_REQUESTED** [review]
Minor change requested, this looks really good!

File: src/content/quickmove.js (hunk 1 above)
Comment (kewisch) [reply src/content/quickmove.js:262]:
```suggestion
          && aFolder.isSpecialFolder(Ci.nsMsgFolderFlags.Archive, false)) {
```

Comment (PaysPlat) [reply src/content/quickmove.js]: OK, I have updated with the constant. Thank you.

**Comment by pspacek** [comment]
Any chance to get this merged? Thanks!

**Comment by kewisch** [comment]
Thank you for updating the pull request, appreciate it!

**Pull request merged**